core/tests/protractor_utils/ExplorationPlayerPage.js

```diff
@@ -0,0 +1,158 @@
+// Copyright 2014 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Page object for the Exploration Player Page,for use in
+ * Protractor tests for playing through an exploration and checking its
+ * contents when carrying out end-to-end testing with protractor.
+ */
+
+var forms = require('./forms.js');
+var general = require('./general.js');
+var interactions = require('../../../extensions/interactions/protractor.js');
+
+var ExplorationPlayerPage = function(){
+  this.restartExploration = function() {
+    element(by.css('.protractor-test-restart-exploration')).click();
+  };
+
+  this.expectExplorationNameToBe = function(name) {
+    expect(
+      element(by.css('.protractor-test-exploration-header')).getText()
```

Comment:
Please save these function calls as variables at the top of the function. You can see examples of what I mean here: https://github.com/brianrodri/oppia/blob/450f6895b41a074088f79a08b371f8ce72dc029e/core/tests/protractor_utils/CreatorDashboardPage.js#L22-L27